spec/integration/runner/variant-reassignment-example-10.spec.js

```diff
@@ -46,7 +46,7 @@ describe('Variant reassignment', () => {
 
   it('merge variants with different sameForAll attributes without blacklist',
     async () => {
-      const reassignment = new VariantReassignment([], logger, {})
+      const reassignment = new VariantReassignment(['brandId'], logger, {})
```

Comment:
I think it's not necessary. Here it's not correct because I changed the constructor a bit, but this should not happen anymore. 88119f5e29ed4563f0ec492cde8eb695763f4147